docs/src/faq/change-date-format.md

```diff
@@ -0,0 +1,9 @@
+# [Changing date format?](@id change_date_format)
+
+The default date format is `yyyy-mm-dd HH:MM:SS`.
+To change that, for example if you want to display milliseconds, you'll have to change the `Formatter` like so:
+
+```julia
+# Changes the formatter of the console handler of the root logger
+gethandlers(getlogger())["console"].fmt = DefaultFormatter("[{date} | {name} | {level}]: {msg}"; date_fmt_string="yyyy-mm-dd HH:MM:SS.sss")
```

Comment:
I don't think we actively want to support this approach, as it seems too easy to break. My recommendation would be to either refactor this package to support passing a dateformat kwarg to `config!` or explicitly mutate `Memento._loggers["root"] = Logger(...)` to reduce how much you're accessing private fields/components. At some point this package will require a refactor to make things less nested.